src/vs/workbench/contrib/debug/browser/rawDebugSession.ts

```diff
@@ -669,7 +669,7 @@ export class RawDebugSession implements IDisposable {
 					...{
 						request: args.request,
 						type: this.dbgr.type,
-						name: this.name
+						name: args.configuration.name || this.name
```

Comment:
Do you want to keep `args.configuration.name` if it's `''`?; should this be `??`?

Reply:
This was on purpose, it didn't seem like someone want an empty name for a debug session. I think debug sessions should have names